model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -644,6 +688,19 @@ class WorkspaceJsonSupport extends JsonSupport {
     }
   }
 
+  implicit object WorkspaceVersionFormat extends JsonFormat[WorkspaceVersion] {
+    override def write(wv: WorkspaceVersion): JsValue = JsString(wv.value)
+
+    override def read(json: JsValue): WorkspaceVersion = json match {
+      case JsString(versionString) => WorkspaceVersions.fromString(versionString) match {
+        case Some(version) => version
+        case None =>throw DeserializationException("unexpected version string")
```

Comment:
Nit: I think there should be a ` ` after the `=>`? (Just given convention I've seen)